### PR TITLE
Week1 ayman

### PR DESCRIPTION
**Pull request by Ayman161803**
Screenshot from 2022-01-29 17-55-31

### CODE REVIEW

File: week1/armstrong.rb [hunk 1]
```diff
@@ -9,5 +9,22 @@
 # and `false` otherwise.
 
 def armstrong_number(n)
-    raise NotImplementedError # Delete this line and start coding here
+    temp = n
+    arm=0
+    noDigits=0
```
Comment (raosush) [inline week1/armstrong.rb:14]: Prefer [snake_case](https://en.m.wikipedia.org/wiki/Snake_case) in Ruby and not camel case

File: week1/armstrong.rb [hunk 2]
```diff
@@ -9,5 +9,22 @@
...
+
+    while n>0 do
+        noDigits=noDigits+1
+        n=n/10
+    end
```
Comment (raosush) [inline week1/armstrong.rb:19]: Ruby is an elegant language and you can it perfectly for such cases.
```ruby
array_of_digits = n.digits
```

**Review by raosush — APPROVED** [review]
Good job! Minor improvements possible. Assignment 1 completed

**Comment by raosush** [comment]
Good job, closing the pull request since changes have been approved

File: week1/armstrong.rb (hunk 1 above)
Comment (Ayman161803) [reply week1/armstrong.rb:14]: Will make sure I use snake_case in the coming assignments.

**Pull request closed without merging**